public/third-party-cookies-check/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Third Party Cookies Test</title>
+  </head>
+
+  <body>
+    <script>
+      const checkCookiesEnable = () => {
+        let isCookieEnabled = window.navigator.cookieEnabled ? true : false
+        if (typeof window.navigator.cookieEnabled == 'undefined' && !isCookieEnabled) {
+          document.cookie = 'testcookie'
```

Comment:
I'd move this to a constant as it is referenced a few times.

Reply:
isCookieEnabled? is updated in https://github.com/gnosis/safe-react/blob/60ea568081fa21ba9bd519e61055ff334961d1b3/public/third-party-cookies-check/index.html#L17

Reply:
Ahh I see. Yep, would be fine. I'll do it